link guide to otel sampling in ingestion mechanisms docs (#18021)
DataDog/documentation: content/en/tracing/trace_pipeline/ingestion_mechanisms.md

| Serverless | `lambda` and `xray`                   | Your traces received from the [Serverless applications][14] traced with Datadog Tracing Libraries or the AWS X-Ray integration. |
| Application Security Management     | `appsec`                            | Traces ingested from Datadog tracing libraries and flagged by [ASM][15] as a threat. |

## Ingestion mechanisms in OpenTelemetry 
`ingestion_reason:otel`

> Depending on your setup with OpenTemetry SDKs (using the OpenTelemetry Collector or the Datadog Agent), you have multiple ways of controlling ingestion sampling. Refer to the guide [ingestion sampling with OpenTelemetry][22] for more details about the options available for sampling at the OTel SDK, OTel Collector and Datadog Agent level in various OpenTelemetry setups.

Reviewer: ```suggestion
Depending on your setup with the OpenTelemetry SDKs (using the OpenTelemetry Collector or the Datadog Agent), you have multiple ways of controlling ingestion sampling. See [Ingestion Sampling with OpenTelemetry][22] for details about the options available for sampling at the OTel SDK, OTel Collector, and Datadog Agent level in various OpenTelemetry setups.
```
